.env-cmdrc.json

```diff
@@ -3,17 +3,17 @@
     "REACT_APP_WSS_URL": "wss://bfx-hf-ui-core.staging.bitfinex.com/ws/",
     "REACT_APP_UFX_PUBLIC_API_URL": "https://api-pub.staging.bitfinex.com",
     "REACT_APP_UFX_API_URL": "https://api.staging.bitfinex.com",
-    "REACT_APP_UFX_WSS_URL": "wss://api.staging.bitfinex.com/ws/2",
```

Comment:
do not remove `REACT_APP_UFX_WSS_URL`, it is required for other ufx comp Book, Trades etc